package.json

```diff
@@ -20,57 +20,58 @@
     "postinstall": "nuxt prepare"
   },
   "dependencies": {
-    "@auth0/auth0-vue": "^2.3.2",
+    "@auth0/auth0-vue": "^2.3.3",
     "@bcc-code/bmm-sdk-fetch": "^7.1.0",
-    "@headlessui/vue": "^1.7.16",
-    "@microsoft/applicationinsights-web": "^3.0.5",
+    "@headlessui/vue": "^1.7.17",
+    "@microsoft/applicationinsights-web": "^3.0.7",
     "@pinia/nuxt": "^0.5.1",
-    "@vueuse/core": "^10.5.0",
+    "@vueuse/core": "^10.7.2",
     "class-variance-authority": "^0.7.0",
     "cross-env": "^7.0.3",
-    "electron-updater": "^6.1.4",
+    "electron-updater": "^6.1.7",
     "pinia": "^2.1.7",
-    "pinia-plugin-persistedstate": "^3.2.0",
-    "vue-draggable-plus": "^0.2.7"
+    "pinia-plugin-persistedstate": "^3.2.1",
+    "vue-draggable-plus": "^0.3.5"
   },
   "devDependencies": {
     "@intlify/eslint-plugin-vue-i18n": "^2.0.0",
     "@nuxt/devtools": "^0.6.4",
-    "@nuxt/kit": "^3.6.1",
-    "@nuxtjs/color-mode": "^3.3.0",
+    "@nuxt/kit": "^3.6.5",
+    "@nuxt/test-utils": "^3.9.0",
+    "@nuxtjs/color-mode": "^3.3.2",
     "@nuxtjs/eslint-config-typescript": "^12.1.0",
-    "@nuxtjs/i18n": "8.0.0-beta.10",
-    "@nuxtjs/plausible": "^0.2.3",
-    "@nuxtjs/tailwindcss": "^6.9.4",
+    "@nuxtjs/i18n": "8.0.0",
+    "@nuxtjs/plausible": "^0.2.4",
+    "@nuxtjs/tailwindcss": "^6.10.4",
     "@pinia-plugin-persistedstate/nuxt": "^1.2.0",
-    "@vitest/coverage-v8": "^0.34.6",
-    "@vue/test-utils": "^2.4.1",
-    "@vueuse/nuxt": "^10.5.0",
+    "@vitest/coverage-v8": "^1.2.0",
+    "@vue/test-utils": "^2.4.3",
+    "@vueuse/nuxt": "^10.7.2",
     "chalk": "^5.3.0",
-    "cypress": "^13.4.0",
-    "electron": "^27.0.3",
-    "electron-builder": "^24.6.4",
-    "eslint": "^8.52.0",
+    "cypress": "^13.6.3",
+    "electron": "^28.1.4",
+    "electron-builder": "^24.9.1",
+    "eslint": "^8.56.0",
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-config-airbnb-typescript": "^17.1.0",
-    "eslint-config-prettier": "^9.0.0",
-    "eslint-plugin-vitest": "^0.3.9",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-vitest": "^0.3.20",
     "glob": "^10.3.10",
     "happy-dom": "^12.10.3",
     "log-symbols": "^6.0.0",
-    "nuxt": "^3.6.1",
-    "nuxt-electron": "^0.6.0",
-    "nuxt-typed-router": "^3.3.2",
-    "nuxt-vitest": "^0.11.2",
-    "prettier": "^3.0.3",
-    "prettier-plugin-tailwindcss": "0.4.1",
-    "stylelint": "^15.11.0",
-    "stylelint-config-recommended-scss": "^13.1.0",
+    "nuxt": "^3.6.5",
```

Comment:
Nuxt 3.9 is already released. Is it too big of a jump from 3.6.1?

Reply:
Can't go to `3.7.0` or further as of now. See https://github.com/nuxt/nuxt/discussions/25257 and the there linked research.

Reply:
The update will be followed up in #230